src/MovieDetails.js

```diff
@@ -2,9 +2,7 @@ import React from 'react';
 import './css/MovieDetails.css'
 
 const MovieDetails = (props) => {
-console.log(props, "<<<<<<<props")
-
-const details = props.details;
+  const details = props.details;
```

Comment:
This works -  but I wanted to let you know that this isn't a pattern you'll see often out in the wild. Instead, you'll see destructuring.